services/catalog/src/simcore_service_catalog/services/director.py

```diff
@@ -0,0 +1,115 @@
+import json
+import logging
+from typing import Dict, Optional
+
+import attr
+from fastapi import FastAPI, HTTPException
+from httpx import AsyncClient, Response, StatusCode
+
+from starlette import status
+
+from ..core.settings import DirectorSettings
+
+
+logger = logging.getLogger(__name__)
+
+
+def setup_director(app: FastAPI) -> None:
+    settings: DirectorSettings = app.state.settings.director
+
+    # init client
+    logger.debug("Setup director at %s...", settings.base_url)
+
+    client = AsyncClient(base_url=settings.base_url)
+    app.state.director_client = client
+
+    # TODO: raise if attribute already exists
+    # TODO: ping?
```

Comment:
do these still require intervention?

Reply:
I guess yes. this comes from copy/paste of the other DB stuff I think.